viewer/src/glsl/base/updateFragmentDepth.glsl

```diff
@@ -17,6 +17,7 @@ void updateFragmentDepth(vec3 p,mat4 projectionMatrix){
 
 void updateFragmentDepth(vec3 p,mat4 projectionMatrix){
   // Extension not available - not much we can do.
```

Comment:
What extension? What are we trying to do here?

Reply:
This is not changed in the current PR. It is supposed to be a no-op if the fragment depth extension for WebGL is not available. Why it currently sets the color to red, I am not sure. Might have been to make it clear that something is wrong. The extension allows us to set the depth buffer to any value we like. Without it, we get the depth from the triangles in the billboards used to create the "impostors" for the different primitives. With it, we can set the depth correctly for spheres, cylinders, etc.